### ISSUE TITLE
Module seems incompatible with Redis

### ISSUE THREAD

**Issue by tlhunter** [charlieduong94/util-promisifyAll#2]
This module doesn't seem to be compatible with the Redis module (which works fine with Bluebird's promisifyAll):

```js
const Redis = require('redis')
const promisifyAll = require('util-promisifyall')
promisifyAll(Redis.RedisClient.prototype)
```

Immediately throws this error:

```js
/home/tlhunter/projects/server/node_modules/redis/index.js:1034                                            
        return this.command_queue.length;                                                                        
                                 ^                                                                               
                                                                                                                 
TypeError: Cannot read property 'length' of undefined                                                            
    at EventEmitter.get (/home/tlhunter/projects/server/node_modules/redis/index.js:1034:34)               
    at _promisifyAllFunctions (/home/tlhunter/projects/server/node_modules/util-promisifyall/index.js:5:25)
    at module.exports (/home/tlhunter/projects/server/node_modules/util-promisifyall/index.js:14:3)        
    at Object.<anonymous> (/home/tlhunter/projects/server/x.js:3:1)                                        
    at Module._compile (module.js:573:30)                                                                        
    at Object.Module._extensions..js (module.js:584:10)                                                          
    at Module.load (module.js:507:32)                                                                            
    at tryModuleLoad (module.js:470:12)                                                                          
    at Function.Module._load (module.js:462:3)                                                                   
    at Function.Module.runMain (module.js:609:10)                                                                
```

**Comment by charlieduong94** [comment]
Thanks for reporting this issue @tlhunter. I'll look into this soon and see what I can do.

**Comment by charlieduong94** [comment]
Hey @tlhunter, sorry for the wait. I published version `1.0.2` which should fix the problem you were seeing with the `redis` module. Please let me know if you encounter any other issues.
